### DISCUSSION TITLE
Filtering on searches

### DISCUSSION THREAD

**Discussion by Lemorz56** [community/community#7959] [Mobile]
Im not sure why you decided to not include filter when searching. For now, using a normal browser on my phone works and looks better then the existing solution. Please add filtering for searches.

**Comment by aconsuegra** [comment]
Hi, thanks a lot for your feedback! Are you referring to the global search that is accessible from Home?

**Reply by Lemorz56** [reply]
Not sure what you mean by home but yeah when doing a search with for example safari on iOS you can access the same filters/sorting as on the website with a computer. You can select a specific language, sort after most starred and so on. I'd like you to bring that on the app as well 😄.
